### DISCUSSION TITLE
Nzb Usenet support,

### DISCUSSION THREAD

**Discussion by tomyprs** [arghyac35/aria-telegram-mirror-bot#184] [Ideas]
add nzb support , it's cool for leech movie :D

**Comment by hsj51** [comment]
> add nzb support , it's cool for leech movie :D

Aria2 (the download client in this repo) doesn't support nzb's, so it's not possible to add support.

**Comment by arghyac35** [comment]
What is nzb?

**Reply by hsj51** [reply]
A simple Google search for **usenet**, **usenet indexers**, **nzb** might help you get the idea. 

In my own words, it is similar to torrent except torrent is peer-to-peer whereas usenet is server-to-peer (but not like direct http). Though usenet is much more than just file sharing.

And .nzb is a file just like .torrent serving similar purpose.

**Reply by hsj51** [reply]
And my knowledge is relatively new (1-2 months) regarding this topic, so it might not be accurate.
